1.hello-world/app.js

```diff
@@ -17,9 +17,16 @@ const port = 3000;
 app.get('/order', (_req, res) => {
     fetch(`${stateUrl}/order`)
         .then((response) => {
-            return response.json();
+            if (!response.ok) {
+                throw "Could not get state.";
+            }
+
+            return response.text();
```

Comment:
response content-type is json so returning response.json() makes sense instead of .text()

Reply:
Actually this fixes the problem when the data is not present (as per issue #90). I tested and it does not change the content-type. I will confirm again before pushing. Since the data is already parsed as JSON in the write operation, returning text() works just fine. The other options would be to use text() only as a fallback when json() fails, or check text() for valid JSON before calling json(). The current change is simpler IMO.

Reply:
Ah ok :). @artursouza do you know what response code returns from dapr for this case? is it 404? I thought we change from 404 to 201 before...

Reply:
It returns 200. Which I think is OK for this.